### DISCUSSION TITLE
How do not automatically restrict full-text search to directory?

### DISCUSSION THREAD

**Discussion by Raphapere** [Zettlr/Zettlr#3247] [Q&A] [answered]
My workflow demands that I can search all folders to create links, not just the current folder.

Whenever I need to search for something I have to use the mouse to remove the restriction to the current folder, this is terrible.

How to search all folders automatically when using "Ctrl+Shift+F" shortcut?

**Comment by nathanlesage** [comment, marked as answer]
I've already noticed that myself, and it annoys me as well. I'm currently thinking about a way to mitigate this.

**Reply by Raphapere** [reply]
I'll wait. Thank you!

**Comment by biva** [comment]
Great, thank you so much! I can share a behaviour that I really appreciate in Evernote to search within notes):
1. Keyboard shortcut to open Evernote (in my case Ctrl + ²) (I do this now with Zettlr thanks to Autohotkey)
2. Evernote opens, with the cursor already in the full-text search box
3. I type a word
4. Evernote displays the results (from all "folders", not only the current one

Would that be possible?
